package.json

```diff
@@ -4,7 +4,8 @@
   "description": "PublicLab.Editor is a general purpose, JavaScript/Bootstrap UI framework for rich text posting, which provides an author-friendly, minimal, mobile/desktop (fluid) interface for creating blog-like content, designed for PublicLab.org",
   "main": "dist/PublicLab.Editor.js",
   "scripts": {
-    "build": "grunt build",
+    "build": "grunt build && ./node_modules/.bin/babel dist -d dist",
```

Comment:
This looks great. Thank you @Shulammite-Aso !!! 

I think perhaps we have to run `grunt babel` before `grunt build` or vice versa, in the travis config:

https://github.com/Shulammite-Aso/PublicLab.Editor/blob/2de7a7f703e7348fe4e320aa0ac7043780932565/.travis.yml#L5-L8

What do you think?

Reply:
Yes @jywarren that sounds right
will just add that now!

Reply:
```
grunt.initConfig({
  babel: {
    options: {
      sourceMap: true,
      presets: ["@babel/preset-env"],
    },
    dist: {
      files: {
        "dist/app.js": "src/app.js",
      },
    },
  },
});

grunt.loadNpmTasks('grunt-babel');

grunt.registerTask("default", ["babel"]);
```

Reply:
I think we need to tell grunt as well that we have to use babel, more details here:
https://babeljs.io/setup#installation